core/trino-spi/src/main/java/io/trino/spi/ptf/DescriptorArgument.java

```diff
@@ -33,20 +33,41 @@
         extends Argument
 {
     public static final DescriptorArgument NULL_DESCRIPTOR = builder().build();
-    private final Optional<Descriptor> descriptor;
+
+    @Nullable
```

Comment:
Why did you change this Optional to a nullable field?

Reply:
It was meant to simplify the code. Reverted.